convert2rhel/checks.py

```diff
@@ -95,56 +97,59 @@ def check_convert2rhel_latest():
     repo_dir = tempfile.mkdtemp(prefix="convert2rhel_repo.", dir=utils.TMP_DIR)
     utils.mkdir_p(repo_dir)
     repo_path = os.path.join(repo_dir, "convert2rhel.repo")
-    store_content_to_file(filename=repo_path, content=repo_content)
+    store_content_to_file(filename=repo_path, content=REPO_CONTENT)
 
     cmd.append("--setopt=reposdir=%s" % repo_dir)
```

Comment:
On CentOS Linux 6:
```
WARNING - Couldn't check if this is the latest version of convert2rhel
repoquery failed 2, Usage: repoquery [options]

repoquery: error: no such option: --setopt
```

We use this option on a couple more places. We weren't testing much on CentOS/OL 6 recently. Please try to find an alternative to the setopt for this old system version.

Reply:
Ok, yeah I can defiantly check this out!

Reply:
I have resolved this in this commit: https://github.com/oamg/convert2rhel/pull/566/commits/41e0a2f2819ad5f3e5d0504f61c1da78bd220a3b